apps/client-frontend/components/Navbar.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import SkillTreeLogo from "../assets/skilltree.svg";
+import Frame from "../assets/frame.svg";
+import Navigation from "../src/components/Navigation";
+
+export default function Navbar() {
+  const [isMobileOpen, setIsMobileOpen] = useState(false);
```

Comment:
I think this part isn't quite working; for me I can open the mobile viewport and I still see the desktop navigation layout.

Reply:
<img width="793" height="867" alt="Screenshot 2025-11-22 at 11 59 21 AM" src="https://github.com/user-attachments/assets/455a5208-cd23-4160-9649-b70cdae4a999" />